### ISSUE TITLE
link contrast - blog

### ISSUE THREAD

**Issue by vavroom** [nicolas-steenhout/main#100]
◦	The link “archive of posts on my old site” has insufficient contrast in light mode.

**Comment by digilou** [comment]
Fixed in https://github.com/nicolas-steenhout/main/commit/e5458464092429ecbb86b68ce583fadbc1e49380
